### PR TITLE
Apply recommendations of docs-lint to README.md for Box Events 0.3.1

### PR DESCRIPTION
**Pull request by djptek**
## What does this PR do?

Applies changes recommended by `docs-lint` to the latest `box_events` integration

## Checklist

~{- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.~

~- [ ] I have verified that all data streams collect metrics or logs.~

- [x] I have added an entry to my package's `changelog.yml` file.

~- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~

## Author's Checklist

```
% docs-lint _dev/build/docs                                                                 (box-docs-lint)integrations
_dev/build/docs/README.md
      9:344-9:347  warning  `DSL` is misspelt; did you mean `DEL`, `ESL`, `ASL`, `DBL`, `DST`, `ISL`?  dsl         retext-spell
       94:3-94:13  warning  `Enablement` is misspelt; did you mean `Ennoblement`?                      enablement  retext-spell
    118:11-118:14  warning  `2FA` is misspelt; did you mean `fA`?                                      2fa         retext-spell
  118:253-118:256  warning  `2FA` is misspelt; did you mean `fA`?                                      2fa         retext-spell
```

- [x] DSL -> refers to Elasticsearch Query DSL
- [x] Enablement -> is copied from target repository documentation
- [x] 2FA -> is copied from target repository documentation & refers to "2 Factor Authentication"

## Related issues

https://github.com/elastic/integrations/pull/4578

## Screenshots

Screenshot 2022-11-08 at 18 18 24
Screenshot 2022-11-08 at 18 19 14
Screenshot 2022-11-08 at 18 20 06
Screenshot 2022-11-08 at 18 20 44

Exported Fields Follow...

### CODE REVIEW

File: packages/box_events/_dev/build/docs/README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ The Box Events integration allows you to monitor [Box](https://app.box.com/). Bo
 
 Use the Box Events integration to ingest the activity logs which are generated each time files are uploaded, accessed, or modified in Box, enabling you to monitor data movement to the cloud. If you have [opted-in to receive additional events](https://developer.box.com/guides/events/event-triggers/shield-alert-events/), the Box Events integration will ingest context-rich alerts on potential threats, such as compromised accounts and data theft, based on anomalous user behavior. Combining this data with other events can lead to the detection of data exfiltration attacks.
 
-Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference `box_events.events` when troubleshooting an issue.
+Then visualize that data in {kib}, create alerts to notify you if something goes wrong, and reference `box_events.events` when troubleshooting an issue.
```
Comment (colleenmcginnis) [inline packages/box_events/_dev/build/docs/README.md:7]:
```suggestion
Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference `box_events.events` when troubleshooting an issue.
```

File: packages/box_events/_dev/build/docs/README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ The Box Events integration allows you to monitor [Box](https://app.box.com/). Bo
...
 
-For example, if you wanted to set up notifications for incoming Box Shield alerts you could verify that this data is being ingested from the `Box Shield Alerts` Dashboard. Then, go to `Alerts and Insights / Rules and Connectors` in the sidebar and set up a Rule using an Elasticsearch Query against index `*box*alert*` with time field `@timestamp` and DSL 
+For example, if you wanted to set up notifications for incoming Box Shield alerts you could verify that this data is being ingested from the `Box Shield Alerts` Dashboard. Then, go to `Alerts and Insights / Rules and Connectors` in the sidebar and set up a Rule using an {es} Query against index `*box*alert*` with time field `@timestamp` and DSL
```
Comment (colleenmcginnis) [inline packages/box_events/_dev/build/docs/README.md:9]:
```suggestion
For example, if you wanted to set up notifications for incoming Box Shield alerts you could verify that this data is being ingested from the `Box Shield Alerts` Dashboard. Then, go to `Alerts and Insights / Rules and Connectors` in the sidebar and set up a Rule using an Elasticsearch Query against index `*box*alert*` with time field `@timestamp` and DSL 
```

File: packages/box_events/docs/README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ The Box Events integration allows you to monitor [Box](https://app.box.com/). Bo
 
 Use the Box Events integration to ingest the activity logs which are generated each time files are uploaded, accessed, or modified in Box, enabling you to monitor data movement to the cloud. If you have [opted-in to receive additional events](https://developer.box.com/guides/events/event-triggers/shield-alert-events/), the Box Events integration will ingest context-rich alerts on potential threats, such as compromised accounts and data theft, based on anomalous user behavior. Combining this data with other events can lead to the detection of data exfiltration attacks.
 
-Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference `box_events.events` when troubleshooting an issue.
+Then visualize that data in {kib}, create alerts to notify you if something goes wrong, and reference `box_events.events` when troubleshooting an issue.
```
Comment (colleenmcginnis) [inline packages/box_events/docs/README.md:7]:
```suggestion
Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference `box_events.events` when troubleshooting an issue.
```

File: packages/box_events/docs/README.md [hunk 4]
```diff
@@ -4,9 +4,9 @@ The Box Events integration allows you to monitor [Box](https://app.box.com/). Bo
...
 
-For example, if you wanted to set up notifications for incoming Box Shield alerts you could verify that this data is being ingested from the `Box Shield Alerts` Dashboard. Then, go to `Alerts and Insights / Rules and Connectors` in the sidebar and set up a Rule using an Elasticsearch Query against index `*box*alert*` with time field `@timestamp` and DSL 
+For example, if you wanted to set up notifications for incoming Box Shield alerts you could verify that this data is being ingested from the `Box Shield Alerts` Dashboard. Then, go to `Alerts and Insights / Rules and Connectors` in the sidebar and set up a Rule using an {es} Query against index `*box*alert*` with time field `@timestamp` and DSL
```
Comment (colleenmcginnis) [inline packages/box_events/docs/README.md:9]:
```suggestion
For example, if you wanted to set up notifications for incoming Box Shield alerts you could verify that this data is being ingested from the `Box Shield Alerts` Dashboard. Then, go to `Alerts and Insights / Rules and Connectors` in the sidebar and set up a Rule using an Elasticsearch Query against index `*box*alert*` with time field `@timestamp` and DSL 
```

**Review by colleenmcginnis — COMMENTED** [review]
@djptek I'm excited to see interest in using `docs-lint`! Unfortunately we can't use these attributes in integration docs. While it is possible to use variables in MDX files, these files are used both by the [docs site](https://docs.elastic.co/integrations) (as MDX files) and in the Kibana UI (as plain Markdown files) so we may never use variables in integration files.

However, this is a good reason for me to dive back into the linter and make sure it's configurable on a per-repo basis so we can enable/disable plugins (like the one for attributes) where necessary.

**Comment by djptek** [comment]
Thanks @colleenmcginnis I've accepted those changes 👍

**Review by efd6 — APPROVED** [review]

**Pull request closed without merging**